tests/director/api/fetch_request_test.go

```diff
@@ -0,0 +1,110 @@
+package api
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/kyma-incubator/compass/components/director/pkg/graphql"
+)
+
+const OpenAPISpec = "https://raw.githubusercontent.com/kyma-incubator/github-slack-connectors/beb8e5b6d8f3a644b8380e667a9376bc353e54dd/github-connector/internal/registration/configs/githubopenAPI.json"
+
+func Test_FetchRequestAddAPIToPackage(t *testing.T) {
```

Comment:
Do we want also test `etchRequestAddApplicationWithAPI`? Create Application along Package and API Definition?